packages/terra-framework-docs/src/terra-dev-site/doc/pills/FilterPills.1.doc.mdx

```diff
@@ -0,0 +1,104 @@
+import { Badge } from 'terra-pills/package.json?dev-site-package';
+
+import AppliedFiltersExample from './example/AppliedFiltersExample?dev-site-example';
+import BasicLabelExample from './example/BasicLabelExample?dev-site-example';
+import CategoryLabelExample from './example/CategoryLabelExample?dev-site-example';
+import TruncatedLabelExample from './example/TruncatedLabelExample?dev-site-example';
+import VisibleLabelExample from './example/VisibleLabelExample?dev-site-example';
+
+import FilterPillsPropsTable from 'terra-pills/src/FilterPills?dev-site-props-table';
+import PillsPropsTable from 'terra-pills/src/subcomponents/_Pill?dev-site-props-table';
+
+<Badge />
+
+# Terra Filter Pills
+
+Pills are generally used to assign a label or keyword to user interface (UI) objects, defining a non-hierarchal group. They are used to visually label UI objects for quick recognition. These labels can be generated either by the system or created by users entering data.
+
+When using a filter workflow, which removes items from a view that do not satisfy specific criteria, Filter Pills can provide indication as to which facet categories and attributes the user can narrow the content by, helping to futher divide content in order to assist users in refining large results.
+
+## What the component is
+
+Filter Pills are comprised of the following:
+
+  - Label ( clickable if truncated to disclose the full label )
+
+  - An optional category to help describe the label
+
+  - An interactive button to remove the filter
+
+## What the component is not
+
+- Although similar, pills and tags have their own use cases and best practices for when they should be used within the UI. See [tags](https://engineering.cerner.com/terra-ui/components/terra-tag/tag/tag) for more information.
+
+- Although the filtering workflow may narrow down content to drive the user's workflow, it should not be used in place of primary or secondary navigation. See primary navigation and secondary navigation for more information.
```

Comment:
should `primary navigation` and `secondary navigation` be hyperlinks?

Reply:
This is language taken from Cerner Design Standards with links internally, and potentially in the future will be a Terra documentation we can link to, but nothing appropriately exists to match yet in open source to add today.